File: Go-Solutions/485.go
package main

func findMaxConsecutiveOnes(nums []int) int {
	count := 0
	maxCount := 0
	for _, v := range nums {
		if v == 1 {
			count++
		} else {
			maxCount = ma(count, maxCount)
			count = 0
		}
	}
	return ma(count, maxCount)
}

func ma(count int, count2 int) int {
	if count > count2 {
		return count
	} else {
		return count2
	}
}

File: Java-Kotlin/src/main/java/array/_485.java
package array;

public class _485 {

    public int findMaxConsecutiveOnes(int[] nums) {
        int count = 0;
        int macCount = Integer.MIN_VALUE;
        for (int num : nums) {
            if (num == 1) {
                count++;
            } else {
                macCount = Math.max(count, macCount);
                count = 0;
            }
        }
        return Math.max(count, macCount);
    }
}
